File: Clases 2018/Clases-C#/ClasePractica11_2018 (update tabla Bugs - asignar bug)/C#_TPS_BugTracker/TPS_BugTracker/GUILayer/Helper/GUIHelper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace TPS_BugTracker.GUILayer.Helper
{
    class GUIHelper
    {
        private static GUIHelper instance; //Unica instancia de la clase
        public static GUIHelper getHelper()
        {
            if (instance == null)
                instance = new GUIHelper();
            return instance;
        }

        public void llenarCombo(ComboBox cbo, Object source, string display, String value)
        {
            cbo.DataSource = source;
            cbo.DisplayMember = display;
            cbo.ValueMember = value;
            cbo.SelectedIndex = -1;
        }
    }
}

File: Clases 2018/Clases-C#/Simulacro_parcial_2_2018 (creacion de bug e historico en transaccion)/VB_TPS_BugTracker/TPS_BugTracker/GUILayer/Helper/GUIHelper.vb
Class GUIHelper
    Private Shared instance As GUIHelper

    Public Shared Function getHelper() As GUIHelper
        If instance Is Nothing Then instance = New GUIHelper()
        Return instance
    End Function

    Public Sub llenarCombo(ByVal cbo As ComboBox, ByVal source As Object, ByVal display As String, ByVal value As String)
        cbo.DataSource = source
        cbo.DisplayMember = display
        cbo.ValueMember = value
        cbo.SelectedIndex = -1
    End Sub
End Class